source/daplink/drag-n-drop/flash_manager.c

```diff
@@ -95,6 +95,10 @@ error_t flash_manager_init(const flash_intf_t *flash_intf)
         return status;
     }
 
+#ifdef FLASH_PARTIAL_ERASE
```

Comment:
The PR #183 adds this as a runtime configuration option. Once that PR is merged then this change will no longer be needed.